rtl/eclrtl/rtlfield.cpp

```diff
@@ -2073,6 +2416,18 @@ size32_t RtlRecordTypeInfo::buildNull(ARowBuilder &builder, size32_t offset, con
     return offset;
 }
 
+size32_t RtlRecordTypeInfo::buildString(ARowBuilder &builder, size32_t offset, const RtlFieldInfo *field, size32_t size, const char *value) const
+{
+    UNIMPLEMENTED;
```

Comment:
Do you ever intend to implement/need? If not should probably be throwUnexpected()

Reply:
(Could actually be useful for my dumpkey code...)